### PR TITLE
Use an enum for color scheme

### PR DESCRIPTION
**Pull request by suve**
This changeset adds a `enum ColorScheme` to the code to replace the magic numbers used for representing the colour scheme. I believe this will make the code a bit more readable.

### CODE REVIEW

File: aha.c [hunk 1]
```diff
@@ -348,14 +354,14 @@ int main(int argc,char* args[])
 			printf("white-space: -pre-wrap; white-space: -o-pre-wrap; word-wrap: break-word;}</style>\n");
 		}
 		printf("</head>\n");
-		if (stylesheet || ! colorshema)
+		if (stylesheet || colorscheme==SCHEME_BLACK)
```
Comment (theZiz) [inline aha.c]: As `!colorsheme` is only true for `colorsheme==0`, the check should be `colorscheme==SCHEME_WHITE`

Comment (suve) [reply aha.c]: Ah, right, I got confused and thought `BLACK` is the default scheme when writing this.

**Comment by suve** [comment]
Changes done.

**Comment by theZiz** [comment]
Thanks!

**Pull request merged**